src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java

```diff
@@ -776,6 +787,89 @@ public void testHaltOnExceptionOff() throws Exception {
         verify(checker, filePath, filePath, expected);
     }
 
+    @Test
+    public void testCheckerProcessCallAllNeededMethodsOfChecks() throws Exception {
```

Comment:
Rename `checks` in this method to `fileSets` and anywhere else it happens in this class when the topic is really file sets.